programs/clearing_house/src/lib.rs

```diff
@@ -709,6 +729,163 @@ pub mod clearing_house {
         Ok(())
     }
 
+    #[access_control(
+        exchange_not_paused(&ctx.accounts.state)
+    )]
+    pub fn settle_lp<'info>(ctx: Context<SettleLP>, market_index: u64) -> Result<()> {
+        let user_key = ctx.accounts.user.key();
+        let user = &mut load_mut!(&ctx.accounts.user)?;
+        let clock = Clock::get()?;
+        let now = clock.unix_timestamp;
+
+        let remaining_accounts_iter = &mut ctx.remaining_accounts.iter().peekable();
+        let market_map =
+            MarketMap::load(&get_writable_markets(market_index), remaining_accounts_iter)?;
+
+        {
+            controller::funding::settle_funding_payments(user, &user_key, &market_map, now)?;
```

Comment:
i think this should just be settling the funding payment for the one market (and in burn/add)